pydicom/tests/test_dataelem.py

```diff
@@ -328,6 +328,39 @@ def test_getitem_raises(self):
         with pytest.raises(TypeError):
             elem[0]
 
+    def test_private_tag_in_repeater_range(self):
+        """Test that a private tag in the repeater range is not handled as
+        repeater tag using Implicit Little Endian transfer syntax."""
+        # regression test for #689
+        ds = Dataset()
+        ds[0x50f10010] = RawDataElement(
+            Tag(0x50f10010), None, 8, b'FDMS 1.0', 0, True, True)
+        ds[0x50f1100a] = RawDataElement(
+            Tag(0x50f1100a), None, 6, b'ACC0.6', 0, True, True)
+        private_creator_tag = ds[0x50f10010]
```

Comment:
Technically, this is a data_elem, not a tag.  Same for private_tag

Reply:
Yes, right - I'm just so used to the term "private tag" that I don't notice that. Corrected.